### PR TITLE
Add html and css for adding footer

### PR DESCRIPTION
**Pull request by divyanshthinks**
This pull request adds a footer section on the portfolio website. This pull request contains the HTML and CSS file for adding footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -204,6 +204,7 @@ <h2 id="footer-heading">CONTACT US</h2>
      </form>
      <br>
```
Comment (ShoiraTa) [inline index.html:205]: - [ ] Kindly make sure that the button has the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template:
Screenshot 2022-06-06 at 21 16 04

- [ ] Kindly center the button as provided in Figma, please make sure that the placeholder text has the same styles as in Figma (font size, family, margins paddings):

Your design  | Figma design
------------- | -------------
Screenshot 2022-06-06 at 21 17 36 | Screenshot 2022-06-06 at 21 17 15

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @divyanshthinks ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 
- Form is fully functional!

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -204,6 +204,7 @@ <h2 id="footer-heading">CONTACT US</h2>
      </form>
```
Comment (youmari) [inline index.html:204]: - [ ] please make sure to implement the effect button on hover and on pressed same as the Figma design as the previous reviewer asked

Comment (youmari) [inline index.html:204]: - [ ] [OPTIONAL] PLEASE try to add some padding to your inputs  in order to match the Figma design 
- yours
- image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @divyanshthinks , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary
✔️ NO linter error
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mukhammadrizooff) [inline index.html:204]: - [ ] [OPTIONAL] Your button should `<button id="footer-button">Send</button> `  but your button text is not like this.
Please make changes!

**Review by mukhammadrizooff — APPROVED** [review]
Hi @divyanshthinks , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
- All linters Passed ✔️ 
- Good implementation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**